src/org/labkey/test/components/ui/grids/DetailTable.java

```diff
@@ -156,6 +159,7 @@ public void clickField(String fieldCaption)
     @Override
     protected ElementCache newElementCache()
     {
+        waitForReady();
```

Comment:
This call is unnecessary. `waitForReady` gets called in `Component.elementCache` before it calls `newElementCache`.
```suggestion
```

Reply:
ah, OK.
I notice that Component.waitForReady(EC ec) is marked as deprecated as of 22.4, I figured that was a signal to find another way (and was, apparently, wrong 😄)